diff --git a/flecsi/topo/unstructured/coloring_utils.hh b/flecsi/topo/unstructured/coloring_utils.hh
index 3a54d12..125a82c 100644
--- a/flecsi/topo/unstructured/coloring_utils.hh
+++ b/flecsi/topo/unstructured/coloring_utils.hh
@@ -567,20 +567,20 @@ color(MD const & md,
     primary.all.insert(primary.all.begin(), p.second.begin(), p.second.end());
 
     for(auto e : p.second) {
-      if(shared.at(p.first).count(e)) {
-        auto d = dependents.at(e);
-        primary.shared.emplace_back(
-          shared_entity{e, {dependents.at(e).begin(), dependents.at(e).end()}});
-      }
-      else {
+      //if(shared.at(p.first).count(e)) {
+      //  auto d = dependents.at(e);
+      //  primary.shared.emplace_back(
+      //    shared_entity{e, {dependents.at(e).begin(), dependents.at(e).end()}});
+     // }
+     // else {
         primary.exclusive.emplace_back(e);
-      } // if
+     // } // if
     } // for
 
-    for(auto e : ghost.at(p.first)) {
-      primary.ghost.emplace_back(ghost_entity{e, e2co.at(e)});
-      primary.all.emplace_back(e);
-    } // for
+    //for(auto e : ghost.at(p.first)) {
+    //  primary.ghost.emplace_back(ghost_entity{e, e2co.at(e)});
+    //  primary.all.emplace_back(e);
+    //} // for
 
     util::force_unique(primary.all);
     util::force_unique(primary.owned);
